### ISSUE TITLE
Citation suffix eats § sign

### ISSUE THREAD

**Issue by Quintus** [jgm/pandoc#7395]
Hi,

I often have to deal with works not cited by page, but instead cited by § and margin number - this is an ordinary case for monographic literature in German Law science. I found pandoc eats my § sign.

Take this input file:

~~~~~~ latex
\documentclass[a4paper,ngerman,11pt]{scrartcl}
\usepackage{babel}
\usepackage[T1]{fontenc}
\usepackage[utf8]{inputenc}
\usepackage[backend=biber,style=verbose-note]{biblatex}

\addbibresource{bibliography.bib}

\begin{document}
\title{Test}
\author{Max Mustermann}
\date{\today}
\maketitle{}
\pagestyle{empty}\thispagestyle{empty}

Dies ist ein Test\footcite[§ 12 Rn. 63]{wandt2014ges-sv}.

\end{document}
~~~~~~

Together with this bibliography file:

~~~~~ bibtex
@Book{wandt2014ges-sv,
author       = {Manfred Wandt},
title        = {Gesetzliche Schuldverhältnisse},
shorttitle = {Gesetzl SV},
year      = {2014},
edition   = {6},
publisher = {Franz Vahlen},
location  = {München}}
~~~~~

When I execute pandoc as follows (`juristische-schulung.csl` can be found [here](https://github.com/citation-style-language/styles/blob/master/juristische-schulung.csl)):

~~~~~~~
pandoc test.tex -C --bibliography=bibliography.bib --csl=juristische-schulung.csl -o result.html
~~~~~~~

Then I receive the following footnote:

~~~~~
1. (Wandt, Gesetzl SV, 6. Aufl. (2014), 12 Rn. 63).↩︎
~~~~~

For the parentheses I opened #7394, so let’s ignore those for the moment. What it should have been is

~~~~~
1. (Wandt, Gesetzl SV, 6. Aufl. (2014), § 12 Rn. 63).↩︎
~~~~~

(note the § sign). I've also tried with markdown input instead, but I was not able to come up with a way to get the § sign rendered. Neither of the following two inputs works either:

~~~~~
Dies ist ein Test [@wandt2014ges-sv, § 12 Rn. 62].
~~~~~

~~~~~
Dies ist ein Test [@wandt2014ges-sv{§ 12 Rn. 62},].
~~~~~

Pandoc version:

~~~~~~
$ pandoc --version
pandoc 2.14.0.2
Compiled with pandoc-types 1.22, texmath 0.12.3, skylighting 0.10.5.1,
citeproc 0.4.0.1, ipynb 0.1.0.1
User data directory: /home/quintus/.local/share/pandoc
Copyright (C) 2006-2021 John MacFarlane. Web:  https://pandoc.org
This is free software; see the source for copying conditions. There is no
warranty, not even for merchantability or fitness for a particular purpose.
~~~~~~

This is on Debian 10 amd64.

**Comment by jgm** [comment]
As a workaround try
```
Dies ist ein Test [@wandt2014ges-sv, {}§ 12 Rn. 62].  
```
This looks like a bug, though.

**Comment by jgm** [comment]
Hm, not sure it's a bug.  Note that if you use the default chicago-author-date style, you get
```
Wandt 2014, sec. 12 Rn. 63
```
So citeproc is recognizing the section symbol as a locator label for a section. It may be that the style you're using doesn't use a locator label in this case?

**Comment by jgm** [comment]
From the style:
```xml
        <else-if type="book">
          <text macro="autor-editor-note" suffix=", "/>
          <choose>
            <if match="none" position="first">
              <group delimiter=" " prefix="(" suffix="), ">
                <text value="o. Fn."/>
                <text variable="first-reference-note-number"/>
              </group>
            </if>
            <else>
              <text variable="title" form="short" suffix=", "/>
              <text macro="edition"/>
            </else>
          </choose>
          <text variable="locator" prefix=", "/>
        </else-if>
```
Note, here you have `<text variable="locator" prefix=", " />`, but no label. I think that if you added `<label variable="locator" form="symbol"/>` before the locator in the style, you'd get what you're expecting...

**Comment by jgm** [comment]
I think this is not a bug.  Pandoc is correctly recognizing the `§` as a locator label, and then doing what the style says to do with locators in this context (not printing the label).  You can work around it in the way I suggested above, by forcing an empty locator `{}` and letting the rest be passed through verbatim as a citation suffix, *or* by modifying the style.

**Comment by Quintus** [comment]
I see, thanks for investigation. I've fiddled with the style a little and it appears you are right. German judicial citations tend to be a hairy beast. The thing is, works cited by page number omit the locator, but those cited by § do not (at least in the journal I am writing for). I will have to see how I can amend the style to honour this; I am rather new to CSL.

> You can work around it in the way I suggested above, by forcing an empty locator 

At least for Markdown input, this works -- thank you. How can I do that with LaTeX input (in a way that doesn't break biblatex)? I tried

~~~~~ latex
Dies ist ein Test\footcite[{}§ 12 Rn. 63]{wandt2014ges-sv}.
~~~~~

(which is left alone by latex/biblatex, because `{}` is effectively a no-op in LaTeX), but it is ignored by pandoc, i.e. it still recognises the § as the locator and thus does not print it with said style.
